apps/daimo-testbed/App.tsx

```diff
@@ -56,7 +56,12 @@ export default function App() {
       "P-256 Public Key, deploy corresponding account by calling createAccount on factory:",
       derPublicKey
     );
-    const account = await DaimoAccount.init(accAddress, signer, false);
+    const account = await DaimoOpSender.init(
+      accAddress,
+      signer,
+      "https://goerli.base.org",
```

Comment:
fine for testbed, for prod this public RPC has low limits

Reply:
hm this is the one we’ve been using in prod for sends/userop ATM. Will pull it out into its own issue to drop this I guess.